caddyhttp/httpserver/context.go

```diff
@@ -57,6 +58,24 @@ func (c Context) Header(name string) string {
 	return c.Req.Header.Get(name)
 }
 
+// Env gets a map of the environment variables.
+func (c Context) Env() map[string]string {
+	osEnv := os.Environ()
+	envVars := make(map[string]string, len(osEnv))
+	for _, env := range osEnv {
+		data := strings.SplitN(env, "=", 2)
+		if len(data) == 2 {
+			envVars[data[0]] = data[1]
+		}
+	}
+	return envVars
+}
+
+// PrintEnv gets the values of all environment variables (for debug purposes).
+func (c Context) PrintEnv() string {
+	return c.Join(os.Environ(), "\n")
+}
```

Comment:
But you can iterate a map in templates with just a couple lines using `.range` - which, IMO, is better because you can control line breaks (if you want `<br>` for example). I still vote we remove this method.